### PR TITLE
Refactor/inline variables

### PR DESCRIPTION
**Pull request by tomchentw**
## Depends on

~#31~

## Description

* Inline several constant code

### CODE REVIEW

File: src/components/ContextMenu/PrefModal.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ import "./PrefModal.css";
 
 const DEFAULT_PREFS = {
   // general
-  //dbcsDetect    : false,
```
Comment (robertabcd) [inline src/components/ContextMenu/PrefModal.js]: I think we should keep this, even there's no way to change this. It's kind of a general feature for BBS.

Comment (tomchentw) [reply src/components/ContextMenu/PrefModal.js]: I just rebased. So did you mean just removing this commit from the history?
https://github.com/robertabcd/PttChrome/pull/32/commits/f19dd642d1cad34ffe02571526d96e17e429c157

Comment (robertabcd) [reply src/components/ContextMenu/PrefModal.js]: Yeah, this would do.

Comment (tomchentw) [reply src/components/ContextMenu/PrefModal.js]: Done.

**Pull request merged**